internal/commands/app/create_test.go

```diff
@@ -230,222 +230,250 @@ func TestAppCreateHandler(t *testing.T) {
-			createdApp = realm.App{
-				GroupID:     groupID,
-				ID:          "456",
-				ClientAppID: name + "-abcde",
-				Name:        name,
-				AppMeta:     meta,
+			cmd := &CommandCreate{
+				inputs: createInputs{
+					newAppInputs: newAppInputs{
+						Name:            "test-app",
+						Project:         "123",
+						Location:        realm.LocationVirginia,
+						DeploymentModel: realm.DeploymentModelGlobal,
+					},
+					Cluster:  tc.cluster,
+					DataLake: tc.dataLake,
+				},
 			}
-			return createdApp, nil
-		}
 
-		var importAppData interface{}
-		rc.ImportFn = func(groupID, appID string, appData interface{}) error {
-			importAppData = appData
-			return nil
-		}
+			assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{Realm: rc, Atlas: tc.atlasClient}))
 
-		ac := mock.AtlasClient{}
-		ac.ClustersFn = func(groupID string) ([]atlas.Cluster, error) {
-			return []atlas.Cluster{{Name: "test-cluster"}}, nil
-		}
+			localApp, err := local.LoadApp(filepath.Join(profile.WorkingDirectory, cmd.inputs.Name))
+			assert.Nil(t, err)
 
-		cmd := &CommandCreate{
-			inputs: createInputs{
-				newAppInputs: newAppInputs{
-					Name:            "test-app",
-					Project:         "123",
+			assert.Equal(t, importAppData, localApp.AppData)
+			assert.Equal(t, realm.App{
+				ID:          "456",
+				GroupID:     "123",
+				Name:        "test-app",
+				ClientAppID: "test-app-abcde",
+				AppMeta: realm.AppMeta{
 					Location:        realm.LocationVirginia,
 					DeploymentModel: realm.DeploymentModelGlobal,
 				},
-				DataSource: "test-cluster"},
-		}
+			}, createdApp)
 
-		assert.Nil(t, cmd.Handler(profile, ui, cli.Clients{Realm: rc, Atlas: ac}))
+			// TODO(REALMC-8262): Investigate file path display options
+			dirLength := len(localApp.RootDir)
+			fmtStr := fmt.Sprintf("%%-%ds", dirLength)
 
-		localApp, err := local.LoadApp(filepath.Join(profile.WorkingDirectory, cmd.inputs.Name))
-		assert.Nil(t, err)
+			var spaceBuffer, dashBuffer string
+			if tc.dataLake != "" {
+				spaceBuffer = "  "
+				dashBuffer = "--"
+			}
 
-		assert.Equal(t, importAppData, localApp.AppData)
-		assert.Equal(t, realm.App{
-			ID:          "456",
-			GroupID:     "123",
-			Name:        "test-app",
-			ClientAppID: "test-app-abcde",
-			AppMeta: realm.AppMeta{
-				Location:        realm.LocationVirginia,
-				DeploymentModel: realm.DeploymentModelGlobal,
+			display := make([]string, 0, 10)
+			display = append(display, "01:23:45 UTC INFO  Successfully created app",
+				fmt.Sprintf("  Info                   "+spaceBuffer+fmtStr, "Details"),
+				"  ---------------------"+dashBuffer+"  "+strings.Repeat("-", dirLength),
+				fmt.Sprintf("  Client App ID          "+spaceBuffer+fmtStr, "test-app-abcde"),
+				"  Realm Directory        "+spaceBuffer+localApp.RootDir,
+				fmt.Sprintf("  Realm UI               "+spaceBuffer+fmtStr, "http://localhost:8080/groups/123/apps/456/dashboard"),
+			)
+			if tc.cluster != "" {
```

Comment:
[sidenote] in general i'm not that big a fan of conditionals around asserts/tests, but with this filepath `display` logic, I think it's fine here for now